state-chain/chains/src/evm.rs

```diff
@@ -24,12 +24,21 @@ use sp_std::{convert::TryFrom, str, vec};
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct EvmCrypto;
 
+// TODO: Since they are different we can probably move this to its own
+// respective files (arb.rs and eth.rs)
 pub mod fees {
-	// TODO: refine these constants.
-	pub const BASE_COST_PER_BATCH: u128 = 50_000;
-	pub const GAS_COST_PER_FETCH: u128 = 30_000;
-	pub const GAS_COST_PER_TRANSFER_NATIVE: u128 = 20_000;
-	pub const GAS_COST_PER_TRANSFER_TOKEN: u128 = 40_000;
+	pub const ETH_BASE_COST_PER_BATCH: u128 = 50_000;
+	pub const ETH_GAS_COST_PER_FETCH: u128 = 30_000;
+	pub const ETH_GAS_COST_PER_TRANSFER_NATIVE: u128 = 20_000;
+	pub const ETH_GAS_COST_PER_TRANSFER_TOKEN: u128 = 40_000;
+
+	// TODO: We might have to be able to update these on the fly. They are 
+	// variable with the L1 calldata prize and L1 calldata size. L1 calldata
+	// price will change in 4844 and L1 calldata size will change as Arb improves.
+	pub const ARB_BASE_COST_PER_BATCH: u128 = 5_200_000;
```

Comment:
Could we have this values so they can be updated on a runtime upgrade? Governance is a bit unnecessary, but we want some way to decrease this amounts when it makes sense.

Reply:
The can already be changed with a runtime upgrade. And we recently introduced a per-chain multplier for gas adjustments. 

We just need to make sure that these values are correct relative to each other.

Reply:
Cool, didn't know that. Then we're good :+1: